website/src/documentation/patterns/bigqueryio.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: section
+title: "BigQuery patterns"
+section_menu: section-menu/documentation.html
+permalink: /documentation/patterns/bigqueryio/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Google BigQuery patterns
+
+The samples on this page show you common patterns for use with BigQueryIO.
+
+## BigQueryIO deadletter pattern
+In production systems, it is useful to implement the deadletter pattern with BigQueryIO outputting any elements which had errors during processing by BigQueryIO into another PCollection for further processing. 
+
+When using `STREAMING_INSERTS`  you can use the `WriteResult` object to access a `PCollection` with the `TableRows` that failed to be inserted into BigQuery. 
```

Comment:
Let's mention retry policies here and mention that that dictates which failures will be added to the deadletter queue and whether we will retry.

Reply:
Fixed.